cmd/loom/loom.go

```diff
@@ -598,14 +600,32 @@ func loadAppStore(cfg *config.Config, logger *loom.Logger, targetVersion int64)
 		if err != nil {
 			return nil, err
 		}
+		iavlStoreRoot := iavlStore.Get(rootKey)
+
 		evmStore, err := loadEvmStore(cfg, iavlStore.Version())
 		if err != nil {
 			return nil, err
 		}
+		evmStoreRoot, evmVersion := evmStore.GetLastSavedRoot(iavlStore.Version())
+		fmt.Println("BEFORE EVM version : %d", evmVersion)
+		fmt.Println("BEFORE IAVL version : %d", iavlStore.Version())
+		if !bytes.Equal(iavlStoreRoot, evmStoreRoot) && evmVersion < iavlStore.Version() {
+			// downgrade iavlStore version
+			fmt.Println("EVM version : %d", evmVersion)
```

Comment:
We should print out some log before trying to load lower version of app.db.
Change this to
```
log.Printf("EVM roots mismatch, evm.db(%d): %X, app.db(%d): %X", evmVersion, evmStoreEvmRoot, appStore.Version(), appStoreEvmRoot)
log.Printf("Try to load app.db at height %d", evmVersion)
```
```